concent_api/core/tests/test_core_views.py

```diff
@@ -310,20 +326,14 @@ def test_receive_should_accept_valid_message(self):
         )
         decoded_response = load(
             response.content,
-            REQUESTOR_PRIVATE_KEY,
-            settings.CONCENT_PUBLIC_KEY,
-        )
-        decoded_inside_message = load(
-            decoded_response.message_task_to_compute,
-            REQUESTOR_PRIVATE_KEY,
-            PROVIDER_PUBLIC_KEY,
+            CONCENT_PRIVATE_KEY,
+            REQUESTOR_PUBLIC_KEY,
         )
-        self.assertEqual(response.status_code, 200)
-        self.assertEqual(new_message.task_id, decoded_inside_message.task_id)
+        self.assertEqual(response.status_code, 200)  # pylint: disable=no-member
```

Comment:
Please check if `pylint: disable=no-member` is really needed here. Last time I checked, pylint was no longer complaining about response fields in tests.